src/tools/crushtool.cc

```diff
@@ -617,6 +617,7 @@ int main(int argc, const char **argv)
       crush.set_item_name(i, "osd." + stringify(i));
     }
 
+    crush.set_type_name(0, "device");
```

Comment:
How about "osd"? That's what the default OSDMap simple generator uses, and this function is also assuming "osd" naming above.